software/tools/oc_find_card

```diff
@@ -357,15 +357,26 @@ while getopts ":vA:C:Vdht:" opt; do
 
                 echo
                 echo "==========================================================================================================="
-                echo "OPAL MSGLOG:"
-                echo "------------"
+		echo "OPAL MSGLOG (from /sys/firmware/opal/msglog file):"
+                echo "--------------------------------------------------"
+		time=$(date +%F-%H%M%S)
+		cp /sys/firmware/opal/msglog "./msglog$time"
+		chmod 666 "./msglog$time"
+		chown $(who am i | awk '{print $1}') "./msglog$time"
+		chgrp $(who am i | awk '{print $1}') "./msglog$time"
```

Comment:
that works, but for the sake of beauty and avoid forking chgrp, chown can do the job too for the group:
myname=$(who am i | awk '{print $1}')
chown $myname:$myname "./msglog$time"

do we really need the "" around the msglog+time file name?
That is all nitpicks, feel free to disregard